skins/laika/src/components/pages/supporters/Supporter.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<div class="accordion-item" v-bind:class="[ isOpen ? 'accordion' : '' ]">
+		<div @click="toggle()">
+			<div v-bind:class="[ isOpen ? 'has-text-primary has-text-weight-bold' : 'accordion-heading', 'icon-inline', 'accordion-title' ] ">
+				<span class="container">{{ content.name }}, {{ content.amount }}</span>
```

Comment:
I've already asked in the Figma document Jan if right-aligned amounts would be possible. He said he designed it this way to be easier to implement, but if right-aligning the amounts is easy, we should do it. How much effort would it be? I really dislike the left-aligned amounts.

Reply:
I actually had that implemented almost completely but since it needed some more fine-tuning and I figured I would need to put in effort for convincing everyone that right-aligned amounts would be better, I threw it away again :grimacing:. Since I was almost there and I remember what I did, I can just re-apply that relatively quickly.

Reply:
@gbirke I have aligned the elements to the right in the latest commit, please check it out.